test/Main.hs

```diff
@@ -1,49 +1,59 @@
 module Main where
 
-import qualified Backends.LLVM as LLVM
-import qualified Backends.Michelson as Michelson
-import qualified Core.Common.Context as Context
-import qualified Core.Common.NameSymb as NameSymb
-import qualified Core.Conv as Conv
-import qualified Core.EAC2 as EAC2
-import qualified Core.Erasure as Erasure
-import qualified Core.IR.Weak as Weak
-import qualified Core.Parser as Parser
-import qualified Core.Typechecker as Typechecker
-import qualified Frontend.Desugar as Desugar
-import qualified Frontend.Parser as Parser
-import qualified FrontendContextualise.Infix.ShuntYard as Shunt
-import qualified FrontendContextualise.Module.Open as Open
-import Juvix.Library hiding (identity)
-import qualified Pipeline
+import Backends.LLVM (backendLLVM)
+import Backends.Michelson (backendMichelson)
+import Core.Common.Context (contextTests)
+import Core.Common.NameSymb (top)
+import Core.Conv (coreConversions)
+import Core.EAC2 (eac2Tests)
+import Core.Erasure (erasureTests)
+import qualified Core.IR.Weak as Weak (top)
+import Core.Parser (coreParser)
+import Core.Typechecker (coreCheckerEval)
+import Frontend.Desugar (allDesugar)
+import Frontend.Golden (contractFiles)
+import Frontend.Parser (allParserTests)
+import FrontendContextualise.Infix.ShuntYard (allInfixTests)
+import FrontendContextualise.Module.Open (openTests)
+import Juvix.Library (IO)
+import Pipeline (tests)
 import qualified Test.Tasty as T
```

Comment:
Can we change all these back?

I really don't want unqualified imports, it is poor that we have redundant names, but we should change the main test name not the qualified itself